koku/api/report/queries.py

```diff
@@ -1012,7 +1013,7 @@ def _order_by(self, data, order_fields):
             if field.startswith("-"):
                 reverse = True
                 field = field[1:]
-            if field in numeric_ordering:
+            if field in numeric_ordering or field in array_ordering:
```

Comment:
Yeah, so to me numeric ordering has to do with numbers. So I felt weird putting the storage_class in that list since it returns an array. Granted looking at it again, `date` isn't numeric, so maybe i'm just over thinking it here. Maybe I should just throw storage_class in the numeric ordering list? What do you think?